### PR TITLE
Adding databento as a source

### PR DESCRIPTION
**Pull request by jmaslek**
As requested by their users, here is a basic introduction of the source for stocks/load and futures/historical

Just test missing and checking for api key ATM

### CODE REVIEW

File: openbb_terminal/stocks/databento_model.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+logger = logging.getLogger(__name__)
+
+# pylint:disable=too-few-public-methods
+
+
+class DataBento(BaseModel):
+    symbol: str
+    exchange: str = "XNAS.ITCH"
+    stype: str = "native"
```
Comment (hjoaquim) [inline openbb_terminal/stocks/databento_model.py:23]: probably the same for `stype`

File: openbb_terminal/sdk_core/models/futures_sdk_model.py [hunk 2]
```diff
@@ -10,8 +10,7 @@ class FuturesRoot(Category):
     Attributes:
         `curve`: Get curve futures [Source: Yahoo Finance]\n
         `curve_chart`: Display curve futures [Source: Yahoo Finance]\n
-        `historical`: Get historical futures [Source: Yahoo Finance]\n
-        `historical_chart`: Display historical futures [Source: Yahoo Finance]\n
```
Comment (hjoaquim) [inline openbb_terminal/sdk_core/models/futures_sdk_model.py:14]: Why remove this endpoint?

**Review by hjoaquim — COMMENTED** [review]
Added a couple of small comments. 
`stocks` worked just fine both on the SDK and the Terminal, but I can't get data for `futures` (maybe wrong interpretation of DataBento symbology?)

File: openbb_terminal/stocks/databento_model.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+from openbb_terminal.rich_config import console
+
+logger = logging.getLogger(__name__)
+
+# pylint:disable=too-few-public-methods
+
+
+class DataBento(BaseModel):
+    symbol: str
+    exchange: str = "XNAS.ITCH"
```
[the remark this answers is not in the document]
Comment (jmaslek) [reply openbb_terminal/stocks/databento_model.py:22]: Why?  They are adding more and this allows it to be passed as a parameter so when we add more down the road we can initialize as DataBento(symbol="AAPL", exchange"OPRA")

File: openbb_terminal/sdk_core/models/futures_sdk_model.py (hunk 2 above)
Comment (jmaslek) [reply openbb_terminal/sdk_core/models/futures_sdk_model.py:14]: added back

File: openbb_terminal/futures/futures_controller.py [hunk 4]
```diff
@@ -165,16 +165,16 @@ def call_historical(self, other_args: List[str]):
...
+            default=(datetime.now() - timedelta(days=2 * 365)),
         )
         parser.add_argument(
             "-e",
             "--end",
             dest="end",
             type=valid_date,
             help="Final date. Default: today",
-            default=datetime.now(),
+            default=datetime.now() - timedelta(days=1),
```
[the remark this answers is not in the document]
Comment (jmaslek) [reply openbb_terminal/futures/futures_controller.py:177]: accidental commit

**Comment by jmaslek** [comment]
> Added a couple of small comments. `stocks` worked just fine both on the SDK and the Terminal, but I can't get data for `futures` (maybe wrong interpretation of DataBento symbology?)

Their historical endpoint does not yet cover up to the last 24 hours (they said soon).  So you likely need to speciify an end date from a few days ago to confirm the date range.

**Comment by hjoaquim** [comment]
@jmaslek I was only able to get a proper response when I added a custom timeout. Do you see any problem with this: https://github.com/OpenBB-finance/OpenBBTerminal/pull/4293/commits/f29a91ba3aabdcf14dc2abfc900c88af40f05a55 ?

**Comment by jmaslek** [comment]
> @jmaslek I was only able to get a proper response when I added a custom timeout. Do you see any problem with this: [f29a91b](https://github.com/OpenBB-finance/OpenBBTerminal/commit/f29a91ba3aabdcf14dc2abfc900c88af40f05a55) ?

For both stocks and futures?  Futures is a large request and it takes a bit of time.  Same with the validation.

**Comment by hjoaquim** [comment]
@jmaslek for `stocks` doesn't seem necessary.

**Comment by jmaslek** [comment]
> @jmaslek for `stocks` doesn't seem necessary.

Yup for futures, it is querying lots of different contracts and takes time.  That is the smart stype.  So its slow, but should not require a timeout.

**Review by hjoaquim — APPROVED** [review]

**Comment by hjoaquim** [comment]
Sorry @jmaslek I misread your comment. This is enough for me to get the data: https://github.com/OpenBB-finance/OpenBBTerminal/pull/4293/commits/fda0c6aa523add831e6a7d075db50062c3addca2

With no "custom" timeout I got the following: 
image

**Pull request merged**